setup.py

```diff
@@ -94,6 +94,12 @@
 }
 
 if 'setuptools' in sys.modules:
+    setup_args['entry_points'] = {
+        'console_scripts': [
```

Comment:
I think this should have been gui_scripts, rather than console_scripts, to launch it without a console window on Windows.